WebApiBackend/WebApiBackend/Helpers/DevelopmentDatabaseSetup.cs

```diff
@@ -55,7 +58,8 @@ public void InitialiseTestDataObjects()
                 DateOfBirth = new DateTime(1984, 02, 09),
                 PhoneNumber = "02243926392",
                 Email = "ZianYangAng@Gmail.com",
-                MedicalInformation = "COVID-19, Extra Chromosome"
+                MedicalInformation = "COVID-19, Extra Chromosome",
```

Comment:
Maybe this has the potential to be insensitive at some point?

Otherwise LGTM

Reply:
True - I'll go change it